vllm/worker/hpu_model_runner.py

```diff
@@ -2402,6 +2472,12 @@ def profile_run(self) -> None:
         )
         return
 
+    def _dummy_run(self, max_num_batched_tokens: int) -> None:
```

Comment:
Can you add a "# noqa" here to skip mypy check, so we can avoid unnecessary changes to model_runner_base and tpu_model_runner?

Reply:
done